PWM_NativePin.h

```diff
@@ -1,6 +1,8 @@
 #pragma once
 
-class PWM_NativePin {
+#include "PWM_Pins.h"
+
+class PWM_NativePin : PWM_Pins {
```

Comment:
indicate that this is like "PWM_Pins". aka adheres to the same interface. Sadly, I can't express this exactly like I want to in c++